src/main.js

```diff
@@ -1 +1,18 @@
-// Write your code here
\ No newline at end of file
+// Write your code here
+    //    1  
+console.log(21 + 21);
+
+    // 2
+
+console.log(21 * 21);
+
+    // 3
+let name = "davit";
+
+console.log("Hello " + name);
+
+    // 4
+let string1 = "Lorem";
+let string2 = "Ipsum";
+
+console.log(string1 + " " + string2);
```

Comment:
სფეისი არ უნდა გქონდეს
```suggestion
console.log(string1 + string2);
```